### PR TITLE
add slides for "Standard Library"

### PR DESCRIPTION
**Pull request by lxxxvi**
I borrowed some utilities from TailwindCSS in order to achieve this.

Screenshot 2020-11-30 at 20 19 42

I liked the result, that's why I added even some more Tailwind CSS classes even though I don't use them (but they're pretty handy).

Also, I decided to put all stdlib examples on 1 slide instead of multiple, what do you think?

Screenshot 2020-11-30 at 20 19 51

### CODE REVIEW

**Review by hharen — COMMENTED** [review]
I like it, it's short and sweet. I'd maybe add one slide just telling that the point of it is that it enables us to use these classes 'for free'. I know it's kind of there but make it really explicit. And also make clear that one doesn't have to install or add anything else, that they're already there, ready to go. Great job ;)

**Comment by lxxxvi** [comment]
@hharen Thanks for the feedback. I figured we could rephrase the "Contents/Purpose" slide, instead of adding another slide... please check.

**Comment by hharen** [comment]
Cool 👍 ;)

**Review by hharen — APPROVED** [review]
Great job!

File: slides/ruby-standard-library/index.html [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+            <li class="fragment fade-in my-4">
+              <b>Networking</b>
+              <div class="text-2xl">Web, E-Mail, Sockets, ...</div>
+            </li>
+            <li class="fragment fade-in my-4">
+              <b>Advanced Math</b>
+              <div class="text-2xl">Prime number generator, Matrices, Complex numbers, ...</div>
+            </li>
+            <li class="fragment fade-in my-4">
+              <b>Date format libraries</b>
```
Comment (nerdinand) [inline slides/ruby-standard-library/index.html:93]:
```suggestion
              <b>Data format libraries</b>
```

I guess you meant that... ;)

**Review by nerdinand — APPROVED** [review]
I agree, it's short and sweet ;)

Maybe at some point we should have some code examples in the slides as well, for example for dates, or something else that's similarly commonly used.

**Comment by lxxxvi** [comment]
@nerdinand Added simple examples for `SecureRandom` and `Time`

**Comment by nerdinand** [comment]
@lxxxvi great stuff! 👍

**Pull request merged**